crates/tuono/.env

```
@@ -0,0 +1,2 @@
+GITHUB_API_BASE_UR=https://api.github.com
```

Sorry I forgot an "L"

```suggestion
GITHUB_API_BASE_URL=https://api.github.com
```